Makefile

```diff
@@ -1,30 +1,29 @@
 ifeq ($(OS),Windows_NT)
   BIN_EXT := .exe
-  SCRIPT_EXT := .ps1
-  SUBSHELL := powershell -ExecutionPolicy Bypass -File
```

Comment:
We used to run powershell to do this. Now, we run a 1-line batch script, where the 1 line is running powershell. Put in a comment explaining why we shouldn't go back to the older, simpler approach. I infer you've learned something subtle about why the straightforward approach is a bad idea, and want to have that knowledge recorded somewhere.

Reply:
Done